### PR TITLE
Mlbapi docstrings

### PR DESCRIPTION
**Pull request by KCNilssen**
### Why

Brings most of the mlbapi.py get functions on the same page using the same styles and requests.

### What

Updates to docstrings as well as a few of the get call structures.

### Tests

Passes tests

### Risk and impact

Medium

### CODE REVIEW

File: mlbstatsapi/mlb_api.py [hunk 1]
```diff
@@ -124,18 +131,99 @@ def get_person(self, player_id, **params) -> Union[Person, None]:
...
+                hydrations
+                # awards
+                currentTeam
+                team
+                rosterEntries
+                relatives
+                transactions
+                social
+                education
+                # stats
```
Comment (Mattsface) [inline mlbstatsapi/mlb_api.py:162]: I don't think you can comment a comment hehe let's just remove stats and awards

File: mlbstatsapi/mlb_api.py [hunk 2]
```diff
@@ -124,18 +131,99 @@ def get_person(self, player_id, **params) -> Union[Person, None]:
...
+        >>> mlb = Mlb()
+        >>> mlb.get_person(660271)
+        Person
+        """
+        if 'stats' in params:
+            return []
+        elif 'awards' in params:
+            return []
+
+        params['personIds'] = person_ids
```
Comment (Mattsface) [inline mlbstatsapi/mlb_api.py:193]: Why plural?

File: mlbstatsapi/mlb_api.py [hunk 3]
```diff
@@ -168,21 +258,55 @@ def get_people_id(self, fullname, sport_id: int = 1,
                     continue
         return player_ids
 
-    def get_teams(self, sport_id: int = 1, **params) -> List[Team]:
+    def get_teams(self, **params) -> List[Team]:
```
Comment (Mattsface) [inline mlbstatsapi/mlb_api.py:261]: since sport is a parameter let's put it up here or move it to other parameters

File: mlbstatsapi/mlb_api.py [hunk 4]
```diff
@@ -988,10 +1427,21 @@ def get_sport_id(self, sport_name: str,
 
         return sport_ids
 
-    def get_league(self, league_id) -> Union[League, None]:
+    def get_league(self, league_id, **params) -> Union[League, None]:
```
Comment (Mattsface) [inline mlbstatsapi/mlb_api.py:1430]: league_id should be a int

File: tests/mock_tests/schedules/test_schedule_mock.py [hunk 5]
```diff
@@ -41,7 +41,7 @@ def tearDownClass(cls) -> None:
 
     def test_get_schedule(self, m ):
         """get_schedule should return schedules for date, starDate, endDate"""
-        m.get('https://statsapi.mlb.com/api/v1/schedule?sportId=1&startDate=04/07/2022&endDate=04/07/2022', json=self.mock_schedule,
+        m.get('https://statsapi.mlb.com/api/v1/schedule?date=04%2F07%2F2022&sportId=1', json=self.mock_schedule,
```
Comment (Mattsface) [inline tests/mock_tests/schedules/test_schedule_mock.py:44]: Does this still work with the special characters?

**Review by Mattsface — COMMENTED** [review]
Just a few minor things

File: mlbstatsapi/mlb_api.py (hunk 2 above)
Comment (KCNilssen) [reply mlbstatsapi/mlb_api.py:193]: The get command is get_persons, Insert personId(s) to return biographical information for a specific player. Format '605151,592450'.

Comment (KCNilssen) [reply mlbstatsapi/mlb_api.py:193]: And we already have a get_person. This get is to grab multiple people at one time, allowing the user to save on making multiple, calls to the api

File: mlbstatsapi/mlb_api.py (hunk 3 above)
Comment (KCNilssen) [reply mlbstatsapi/mlb_api.py:261]: Agree

File: mlbstatsapi/mlb_api.py (hunk 4 above)
Comment (KCNilssen) [reply mlbstatsapi/mlb_api.py:1430]: Agree

File: tests/mock_tests/schedules/test_schedule_mock.py (hunk 5 above)
Comment (KCNilssen) [reply tests/mock_tests/schedules/test_schedule_mock.py:44]: Yup it does, but let me go and change that to slashes

File: mlbstatsapi/mlb_api.py (hunk 2 above)
Comment (Mattsface) [reply mlbstatsapi/mlb_api.py:191]: Oh nice so you can pass two people IDs to the people endpoint?

**Review by Mattsface — APPROVED** [review]
LGTM

**Pull request merged**